packit_service/worker/handlers/comment_action_handler.py

```diff
@@ -76,26 +70,16 @@ def add_to_comment_action_mapping_with_name_inner(
     return add_to_comment_action_mapping_with_name_inner
 
 
-class CommentActionHandler(Handler):
+class CommentActionHandler(JobHandler):
     type: CommentAction
 
     def __init__(
         self,
-        config: ServiceConfig,
-        event: Union[
-            PullRequestCommentGithubEvent,
-            IssueCommentEvent,
-            PullRequestCommentPagureEvent,
-        ],
-        job: JobConfig,
+        package_config: PackageConfig,
+        job_config: Optional[JobConfig],
+        event: dict,
```

Comment:
I'll be honest that I don't like this - passing dict, that will make things harder to debug since it's dict and anything can be inside, not sure if we could pass real objects here which have defined the (de-)serialization process

this comment is just my observation, no need to take any action

Reply:
@lbarcziova you created a new class for those metadata, didn't you?

Cannot be that enhanced to server also there?

I would like to avoid dicts for the same reason as I wrote in build-helpers: typing, debugging,...